Themis/Themis.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Workspace
-   version = "1.0">
-</Workspace>
```

Comment:
Does anyone know if this should be in Git?

Reply:
I don‘t know😬